Makefile

```diff
@@ -31,7 +31,7 @@ SIMAVR-OBJ := obj-${shell $(CC) -dumpmachine}
 OBJ-PREFIX := obj
 OBJ := ${OBJ-PREFIX}/${SIMAVR-OBJ}
 
-LDFLAGS += -L${simavr}/${SIMAVR-OBJ} -lsimavr
+LDFLAGS += -L${simavr}/${SIMAVR-OBJ} -lsimavr -lelf
```

Comment:
If I recall correctly libelf should be already linked into libsimavr so it should not be required here. Do you actually get an error when linking without -lelf here?

Reply:
Yes, I get linking error without -lelf on Ubuntu 18.04:
```ShellSession
LD obj/obj-x86_64-linux-gnu/ssd1306_virt.o
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_nextscn'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `gelf_getshdr'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_getdata'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `gelf_getsym'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_begin'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_end'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_strptr'
./simavr/simavr/obj-x86_64-linux-gnu/libsimavr.so: undefined reference to `elf_version'
collect2: error: ld returned 1 exit status
```

Reply:
See also https://github.com/dxxb/sim-arduboy/issues/6

Reply:
I just tested this and indeed `-lelf` is required.